index.js

```diff
@@ -0,0 +1,67 @@
+module.exports = {
+  "rules": {
+    "at-rule-empty-line-before": [ 2, "always", {
+      except: ["blockless-group"],
+    } ],
+    "at-rule-no-vendor-prefix": 0,
```

Comment:
Would the consumer not be using the distributed version of the CSS with the comments stripped? I think you can add whatever comments you like to your lib versions to aid in the development of that lib.